superchain/superchain.go

```diff
@@ -106,7 +106,8 @@ type ChainConfig struct {
 	SequencerWindowSize uint64 `toml:"seq_window_size"`
 
 	// Optional feature
-	Plasma *PlasmaConfig `toml:"plasma,omitempty"`
+	Plasma         *PlasmaConfig `toml:"plasma,omitempty"`
+	GasPayingToken *Address      `toml:"gas_paying_token, omitempty"` // Just metadata, not consumed by downstream OPStack software
```

Comment:
```suggestion
	GasPayingToken *Address      `toml:"gas_paying_token,omitempty"` // Just metadata, not consumed by downstream OPStack software
```
It might need to be this way. It's the usual way at least.

Reply:
[96ebb0a](https://github.com/ethereum-optimism/superchain-registry/pull/399/commits/96ebb0a0a6b2dc505e805d956ddbd6ad3d680997)